### PR TITLE
add otel in focus july

### PR DESCRIPTION
**Pull request by austinlparker**
n/t

---

**Preview**: https://deploy-preview-3129--opentelemetry.netlify.app/blog/2023/otel-in-focus-07/

### CODE REVIEW

**Review by cartermp — COMMENTED** [review]
LGTM modulo the textlint stuff (that I disagree with, hmmm, lemme se ehow to change that)

**Comment by austinlparker** [comment]
what's weird is that npm fix stuff isn't showing i have any errors in this file...

**Comment by cartermp** [comment]
Doin' here: https://github.com/open-telemetry/opentelemetry.io/pull/3130

File: content/en/blog/2023/otel-in-focus-07.md
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:42]:
```suggestion
The major addition in 0.82 is the ability to export collector telemetry via
```

File: content/en/blog/2023/otel-in-focus-07.md
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:66]:
```suggestion
includes updates to OTLP and log support.
```

File: content/en/blog/2023/otel-in-focus-07.md
Comment (svrnm) [inline content/en/blog/2023/otel-in-focus-07.md:27]:
```suggestion
the configuration of  `TracerProvider` and `MeterProvider`.
```

**Review by svrnm — COMMENTED** [review]
some minor suggestions

**Review by chalin — APPROVED** [review]
A few minor copyedits, otherwise LGTM once the textliner is happy.

Maybe for now we can just use "bugfix" and fix the rule after?

File: content/en/blog/2023/otel-in-focus-07.md
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-07.md:29]:
```suggestion
##### [Semantic Conventions](/docs/specs/semconv/)
```

**Review by chalin — APPROVED** [review]
Link fix is needed, otherwise, LGTM

**Comment by chalin** [comment]
Woohoo, all checks are green!
Any other comments @svrnm @cartermp. If not, let's get this merged!?!

**Review by svrnm — APPROVED** [review]

**Pull request merged**
